qa/tasks/cephfs/test_misc.py

```diff
@@ -65,7 +65,7 @@ def test_fs_new(self):
 
         get_pool_df = self.fs.get_pool_df
         self.wait_until_true(
-                lambda: get_pool_df(self.fs.metadata_pool_name)['objects'] > 0,
+                lambda: get_pool_df(self.fs.metadata_pool_name, timeout=30)['objects'] > 0,
```

Comment:
If you use a wait_until_true here, you can leave get_pool_df as a naive function and avoid rewriting the `while True` stuff